.github/workflows/run_pr_tests.yml

```diff
@@ -381,3 +391,35 @@ jobs:
         toolchain_file: "scripts/../platform/arm-linux/aarch64-toolchain.cmake"
         extra_flags: -DCA_BUILTINS_TOOLS_DIR=${{ github.workspace }}/llvm_install_native/bin
 
+  # Based on: mr-ubuntu-gcc-x86_64-clik
+  run-ubuntu-gcc-x86_64-clik:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 60
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.1.0
+    - name: setup-ubuntu
+      uses: ./.github/actions/setup_build
```

Comment:
I don't think clik requires llvm so maybe we can get rid of that part - it's also so fast I'm not sure we need the cache, but if we do we should be consistent below.

Reply:
we also don't need vulkan sdk, I'm wondering if we should not bother calling setup_ubuntu and just install ninja if needed (or add a few more args so we get the basics).

Reply:
I wondered about that for this job. Both clik jobs are green now so I'll investigate a minimal preamble.

Reply:
Minimal preamble used.